src/actions/RestActions.js

```diff
@@ -573,15 +575,11 @@ export const placeGuestCartOrder = (cartId, payment) => {
       } else {
         data = await magento.guest.placeGuestCartOrder(cartId, payment);
       }
-      if (!data.message) {
-        dispatch({ type: MAGENTO_PLACE_GUEST_CART_ORDER, payload: data });
-      } else {
-        const message = magento.getErrorMessageForResponce(data);
-        dispatch({ type: MAGENTO_ERROR_MESSAGE_CART_ORDER, payload: message });
-      }
+      dispatch({ type: MAGENTO_PLACE_GUEST_CART_ORDER, payload: data });
       dispatch({ type: UI_CHECKOUT_CUSTOMER_NEXT_LOADING, payload: false });
     } catch (error) {
       console.log(error);
+      dispatch({ type: MAGENTO_ERROR_MESSAGE_CART_ORDER, payload: error });
```

Comment:
Now error will be handled in catch clause no need to manually check for `data.message`